### PR TITLE
Fix commit hashes refering to the wrong repository

### LINKED ISSUE #35: [2H] Commit hashes in generated page do not relate to the correct repository
As [@FlorianDussable said](https://github.com/chocoteam/PerformanceAction/pull/34#discussion_r650992559), the commit hashes passed to the program are the hashes in `chocoteam/PerformanceAction`, not `chocoteam/choco-solver`.

### PR DESCRIPTION
**Pull request by RemiBardon**
Closes #35.

### CODE REVIEW

File: src/main.py [hunk 1]
```diff
@@ -48,14 +48,16 @@ def shared_main(input_file_path: str, hashref: str, hashcomp: str, ref_content,
         test_folder_path=comp_content["metadata"]["testFolderPath"],
         page_title=comp_content["metadata"]["pageTitle"],
         page_description=comp_content["metadata"]["pageDescription"],
+        code_commit=comp_content["metadata"]["codeCommit"],
```
Comment (FlorianDussable) [inline src/main.py:51]: I wouldve called it "commitHash" instead but it's no big deal

Comment (FlorianDussable) [inline src/main.py:51]: Also i would've put the ref commit hash in metadata aswell, instead of settings. To discuss

File: src/models.py [hunk 2]
```diff
@@ -24,27 +24,31 @@ def __init__(self, file_path: str, exit_diff: Diff, diffs: List[Diff]):
...
             page_description (str): Description of the generated Hugo page
+            code_commit (str): Hash of the commit used to run tests (in the main repository, where the code change originated)
         """
         self.test_folder_path = test_folder_path
         self.page_title = page_title
         self.page_description = page_description
+        self.code_commit = code_commit
 class PageGenSettings:
-    def __init__(self, test_output_metadata: TestOutputMetadata, repository_url: str, similar_percent_limit: float = 1):
+    def __init__(self, test_output_metadata: TestOutputMetadata, ref_code_commit: str, repository_url: str, similar_percent_limit: float = 1):
```
Comment (FlorianDussable) [inline src/models.py:41]: See comment on main.py

File: src/main.py (hunk 1 above)
Comment (RemiBardon) [reply src/main.py:51]: It's a commit in the main code repository, I wanted to make it explicit.

File: src/models.py (hunk 2 above)
Comment (RemiBardon) [reply src/models.py:41]: See comment on it ;)

**Pull request merged**